scripts/import/import_EPMC.pl

```diff
@@ -153,75 +156,83 @@ sub import_citations{
+                if(defined $pub_pmid && $done_list->{$rsid}{$pub_pmid}){  
+                  my $var_id = $v->dbID();  
+                  $pub_ad->update_citation_data_source($source_attrib_id, $var_id, $pub_pmid); 
+                } 
             }
             else{
                 no warnings ;
                 ### write file of variants not found in this species to use as input file for next
                 if($type eq "EPMC"){
-                  print $not_found "$rsid,$data->{$pub}->{pmcid},$data->{$pub}->{pmid}\n"; 
+                  print $not_found "$rsid,$data->{$pub}->{pmcid},$pub_pmid\n"; 
                 }
                 elsif($type eq "UCSC") {
-                  print $not_found $rsid ."\t". $data->{$pub}->{pmid} ."\t-\t". $data->{$pub}->{doi} ."\t". $data->{$pub}->{title}."\t". $data->{$pub}->{authors} ."\t". $data->{$pub}->{year} ."\t".  $data->{$pub}->{ucsc} . "\n";
+                  print $not_found $rsid ."\t". $pub_pmid."\t-\t". $data->{$pub}->{doi} ."\t". $data->{$pub}->{title}."\t". $data->{$pub}->{authors} ."\t". $data->{$pub}->{year} ."\t".  $data->{$pub}->{ucsc} . "\n";
                }
                use warnings ;
-            }
+            }  
         }
-
-
-        ## don't enter publication if there are no variants found for this species
-        next unless defined $var_obs[0] ;
-        
+    
+        # don't enter publication if there are no variants found for this species
+        next unless defined $var_obs[0];        
         
-      
         ### Check if publication already known & enter if not  
         my $publication;
-
+        
         ## looking up missing data from EPMC before redundancy check
         my $ref = get_publication_info_from_epmc($data, $pub, $error_log);
-	
+        
         my $title = (defined $ref->{resultList}->{result}->{title} ? $ref->{resultList}->{result}->{title}  : $data->{$pub}->{title});
         next unless defined $title;       
-        next if $title =~/Erratum/i; 
- 
+        next if ($title =~/Erratum/i || $title =~/Errata/i || $title =~/Not Available/i); 
+        # Delete brackets from title, e.g. [title]. 
+        if($title =~ /^\[ ?[A-Za-z]{2}/){
+          $title =~ s/^\[//;
+          $title =~ s/\]\.//; 
+        }  
+
         ## save ids 
         my $pmid   = $ref->{resultList}->{result}->{pmid}   || $data->{$pub}->{pmid};
         my $pmcid  = $ref->{resultList}->{result}->{pmcid}  || undef;
         my $doi    = $ref->{resultList}->{result}->{DOI}    || $data->{$pub}->{doi};
-
+        
         ## try looking up on doi first
         $publication = $pub_ad->fetch_by_doi( $doi )     if defined $doi;
         ## then PMID
         $publication = $pub_ad->fetch_by_pmid( $pmid )   if defined $pmid && ! defined $publication;
         ## then PMCID   
         $publication = $pub_ad->fetch_by_pmcid( $pmcid ) if defined $pmcid && ! defined $publication;
-
-
+        
+        
```

Comment:
Can you avoid uneccessary whitespace additions.

Reply:
I deleted those ones but it looks worse with all the other spaces.